components/runtime/spidermonkey/Makefile

```diff
@@ -0,0 +1,73 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"). You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2017 Andreas Grueninger, Grueninger GmbH, (grueni). All rights reserved. 
+#
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=	          spidermonkey
+COMPONENT_VERSION=        1.8.5
+COMPONENT_SUMMARY=        Javascript engine
+COMPONENT_PROJECT_URL=    https://developer.mozilla.org/en-US/docs/Mozilla/Projects/SpiderMonkey
+COMPONENT_FMRI=	          runtime/spidermonkey
```

Comment:
library/spidermonkey (as it's not an interpreter), please move the component correspondingly